pcrmc/database.py

```diff
@@ -54,6 +54,17 @@ def write_contacts(self, contact_list: List[Dict[str, Any]]) -> DBResponse:
         except OSError:
             return DBResponse(contact_list, DB_WRITE_ERROR)
 
+    def get_new_contact_id(self, config_file: Path) -> int:
+        config_parser = configparser.ConfigParser()
+        config_parser.read(config_file)
+        id = int(config_parser["General"]["RunningID"])
+        config_parser["General"]["RunningID"] = str(id + 1)
+        try:
+            with config_file.open("w") as file:
+                config_parser.write(file)
+        except OSError:
+            return -1
```

Comment:
Define specific error in __init__.py and use that instead of -1

Reply:
done